### ISSUE TITLE
[HELP] OSKR activation fails

### ISSUE THREAD

**Issue by Schattenruf** [kercre123/wire-pod#275]
**OSKR but cannot activate**

**What are you running wire-pod on?**
  - [Hardware]: (Beelink BT3 Pro)
  - Debian

**Are you setting up a prod (regular) bot or a dev/OSKR (unlocked) bot?**
OSKR Bot with custom firmware from wire.my.to (shows now 1.9.0oskr)
**What steps have you tried?**
Deleting user data, upload ssh certificate (went through flawlessly), skipped onboarding process from custom firmware
Pairing in Bluetooth mode functions but it will crash and I just see my wifi network and a loop of death. Vector will show me his eyes immediately.
If I use the old keriganc epod setup page, it will go through but I get an error when activating.
New Page:
![image](https://github.com/kercre123/wire-pod/assets/12720317/25be4973-3860-4910-b68e-f3bb3bbd317e)

old Page:
![image](https://github.com/kercre123/wire-pod/assets/12720317/fae20869-fb67-4101-8e8d-f0cb204f85ae)

So how can I activate my vector again?

**Comment by kercre123** [comment]
When you were setting up wire-pod, did you choose the "Escape Pod" config or "IP Address" config? Is the Beelink running wire-pod connected to Wi-Fi or Ethernet?

**Comment by Schattenruf** [comment]
> When you were setting up wire-pod, did you choose the "Escape Pod" config or "IP Address" config? Is the Beelink running wire-pod connected to Wi-Fi or Ethernet?

After the install of wirepod I got into the settings and chose the Bot Setup and there the IP address. Prior to that there was no question about how to connect (at least as far as I remember).

WiFi or ethernet?: for the test I tried both but with no success.

**Comment by kercre123** [comment]
What is the IP address of the machine? And could you SSH into the bot, run `cat /anki/data/assets/cozmo_resources/config/server_config.json`, then send the output here.

**Comment by Schattenruf** [comment]
Ok I will do this on Monday. Not at home this weekend ;)

**Comment by edgera** [comment]
I also encountered this issue on the keriganc page: new https://wpsetup.keriganc.com/html/main.html. 
This string handling behavior from rts.js seems to be the cause of the page hanging.

![cause](https://github.com/kercre123/wire-pod/assets/8787587/eea84ac3-5457-403a-93da-947c701519b4)

I was able to successfully complete activation by locally replacing this js with an arbitrary constant immediately on initial page load. Then I was able to pair with vector and proceed as normal.

![fix](https://github.com/kercre123/wire-pod/assets/8787587/31206eb0-5ee7-4266-b579-302b55ef3403)
